src/geodedic.rs

```diff
@@ -38,6 +38,17 @@ const SEMI_MINOR_AXIS: f64 = SEMI_MAJOR_AXIS * (1.0 - FLATTENING);
 //     = 2 * f - f^2
 const ECCENTRICITY_SQ: f64 = 2.0 * FLATTENING - FLATTENING * FLATTENING;
 
+// Altitude range for which we guarantee From<Ecef> for Wgs84.
+const ECEF_TO_WGS84_MIN_ALTITUDE_M: f64 = -10_000.0;
+const ECEF_TO_WGS84_MAX_ALTITUDE_M: f64 = 50_000.0;
```

Comment:
In the old tests, we had the max be 40k (-10k + 50k), but if this still passes the tests, then this new max is fine (and better aligns with the top of the stratosphere anyway!).